packages/tools/fluid-runner/src/index.ts

```diff
@@ -3,5 +3,6 @@
  * Licensed under the MIT License.
  */
 
-export * from "./codeLoaderBundle";
-export * from "./exportFile";
+export { ICodeLoaderBundle, IFluidFileConverter } from "./codeLoaderBundle";
```

Comment:
Curiosity, what's the use case for exporting these out of the package? I just skimmed the repo but it doesn't seem that we have anything there that uses them outside of fluid-runner itself, but you put these here and not the rest of the things exported in `codeLoaderBundle`... so wondering who/what uses them outside this package?

And similar for the exports from `exportFile`.

Reply:
The idea of the fluid-runner `exportFile` functionality is that consumers can access it at any layer they want. They can choose to consumer it as a library and handle most things themselves, import and run it directly from the command line providing their own bundled implementation for `ICodeLoaderBundle` and `IFluidFileConverter`, or create a wrapper around the runner app to pass in a `IFluidFileConverter` implementation for bundling everything together.

As for the specific exports of `./codeLoaderBundle`, the interfaces are the only important thing to external consumers. The functions around validating an object type can be considered an implementation detail of the fluid-runner.